config/settings.yml

```diff
@@ -1644,6 +1644,10 @@ vaos:
     mock: <%= ENV['vaos__eps__mock'] %>
     redis_token_expiry: 3500
     scopes: care-nav
+  referral:
+    encryption:
+      hex_iv: <%= ENV['vaos__referral__encryption__hex_iv'] %>
+      hex_secret: <%= ENV['vaos__referral__encryption__hex_secret'] %>
```

Comment:
These don't appear to be set for any environment in Param Store. Does these need to be added?

Reply:
Yes, @lee-delarm6 is doing that today as a separate ticket.